build/version/version.go

```diff
@@ -0,0 +1,181 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package version provides information about FerretDB version and build configuration.
+//
+// # Required files
+//
+// The following generated text files may be present in this (build/version) directory during building:
+//   - version.txt (required) contains information about the FerretDB version in a format
+//     similar to `git describe` output: `v<major>.<minor>.<patch>`.
+//   - commit.txt (optional) contains information about the source git commit.
+//   - branch.txt (optional) contains information about the source git branch.
+//
+// # Go build tags
```

Comment:
`unix` build tag is not described. Is it on purpose?

Reply:
We don't document standard Go build tags like `unix` or `race`, only ours